recipe_scrapers/_schemaorg.py

```diff
@@ -57,6 +59,16 @@ def __init__(self, page_data, raw=False):
                     if key:
                         self.people[key] = person
 
+        # Extract ratings data
+        for syntax in SYNTAXES:
+            syntax_data = data.get(syntax, [])
+            for item in syntax_data:
+                rating = self._find_entity(item, "AggregateRating")
+                if rating:
```

Comment:
At some point we could optionally reduce the filesize by a few lines by using the [walrus operator](https://docs.python.org/3/reference/expressions.html) here -- it looks like I missed one of out two opporunties to do that [previously here](https://github.com/hhursev/recipe-scrapers/pull/913/files#diff-090f49f080e792bcb15643b731714bd07fe0ab25e9eaf92c6f15e53667e88c83R58-R59).  It should be safe to do that because this library [requires Python 3.8 or greater](https://github.com/hhursev/recipe-scrapers/blob/69d334f79bc456ee43ad94bd02bc75bba7c9d30f/pyproject.toml#L28).